src/utils.ts

```diff
@@ -184,15 +184,16 @@ export async function buildCeramic(configObj, ipfs?: IpfsApi): Promise<CeramicAp
       }
     }
     const [modules, params] = await Ceramic._processConfig(ipfs, ceramicConfig)
+    const ceramic = new Ceramic(modules, params)
+    const did = await createDid(seed)
+    ceramic.did = did
     if (configObj.s3StateStoreBucketName) {
       const bucketName = `${configObj.s3StateStoreBucketName}${S3_DIRECTORY_NAME}`
-      const s3StateStore = new S3StateStore(bucketName, loggerProvider.getDiagnosticsLogger())
-      modules.pinStoreFactory.setStateStore(s3StateStore)
+      const s3Store = new S3Store(bucketName, null, configObj.network)
```

Comment:
is it safe to pass `null` for the `endpoint` here?  If not, what should be used here?

Reply:
Yes, `undefined` and `null` are safe, just tested it manually. Sorry, I'll publish a PR to make this param optional in `S3Store`'s contructor.

If the integration tests are passing, please merge.